apps/docs/.vitepress/plugins/snippetPlugin.ts

```diff
@@ -3,10 +3,11 @@ import path from 'path';
 import fs from 'fs';
 import { ErrorCode, FuelError } from '@fuel-ts/errors';
 import { RuleBlock } from 'markdown-it/lib/parser_block';
-import { extractImports } from './utils/extractImports';
+import { extractImports, validateSnippetContent } from './utils/extractImports';
 
 // Regex to match import comments
 export const IMPORT_REGEXP = /\/\/ #import \{(.+)\};$/;
+export const IMPORT_START_REGEXP = /\/\/ #import/;
```

Comment:
This can be put into `validateSnippetContent` because that's the only place where it's used.

Reply:
It could be - but they're related constants so I'll leave it here.